app/src/main/java/org/fossasia/pslab/activity/MainActivity.java

```diff
@@ -185,7 +184,7 @@ private Fragment getHomeFragment() throws IOException {
             case 1:
                 return HomeFragment.newInstance(ScienceLabCommon.scienceLab.isConnected(), ScienceLabCommon.scienceLab.isDeviceFound());
             case 2:
-                return SettingsFragment.newInstance();
+                /**/
```

Comment:
break or return from here. There is no point in checking other cases once it's here isn't it?